.github/workflows/ci.yaml

```diff
@@ -111,13 +111,16 @@ jobs:
       - uses: actions/checkout@v4
       - name: "Install Rust toolchain"
         run: rustup show
-      - name: "Install cargo insta"
+      - name: "Install mold"
+        uses: rui314/setup-mold@v1
+      - name: "Install cargo nextest"
         uses: taiki-e/install-action@v2
         with:
-          tool: cargo-insta
+          tool: cargo-nextest
       - uses: Swatinem/rust-cache@v2
       - name: "Run tests"
-        run: cargo insta test --all --all-features --unreferenced reject
```

Comment:
Are we fine with losing the `--unreferenced reject` here?

Reply:
Yes

Reply:
I'm not :(

Reply:
It’s not possible (IIUC) to preserve this with Nextest, and I just don’t think it’s the right tradeoff to choose the test runner based on this. It’s something we can run locally every once in a while.

Reply:
Oh, it looks like you _can_ use Nextest with Insta? I'm confused, I'd never looked into it but there was a conversation elsewhere suggesting that it wasn't possible to preserve this with Nextest \cc @zanieb.